administrator/language/de-DE/plg_fields_menuitem.ini

```
@@ -4,6 +4,6 @@
 ; License GNU General Public License version 2 or later; see LICENSE.txt
 ; Note : All ini files need to be saved as UTF-8
 
-PLG_FIELDS_MENUITEM="Fields - Menu Item"
-PLG_FIELDS_MENUITEM_LABEL="List of Menu Items (%s)"
-PLG_FIELDS_MENUITEM_XML_DESCRIPTION="This plugin lets you create new fields of type 'menuitem' in any extensions where custom fields are supported."
+PLG_FIELDS_MENUITEM="Felder - Menüeintrag"
+PLG_FIELDS_MENUITEM_LABEL="Liste von Menüeinträgen (%s)"
+PLG_FIELDS_MENUITEM_XML_DESCRIPTION="Dieses Plugin ermöglicht das Erstellen von Feldern des Typs 'menuitem' in Erweiterungen, die eigene Felder unterstützen.
```

Vorschlag: 
```suggestion
PLG_FIELDS_MENUITEM_XML_DESCRIPTION="Mit diesem Plugin können in allen Erweiterungen, die benutzerdefinierte Felder unterstützen, neue Felder vom Typ 'menuitem' erstellt werden."
```
fehlendes " am Ende

---

Ist halt vom Wording wie alle anderen Plugins.

Vorschlag wir mergen das und du machst übers Wochenende einen neuen PR für alle field Plugins?

---

@tecpromotion, @zero-24, @heelc29 
Bevor ich mich an die Arbeit mit den 32 Dateien mache, möchte ich gerne wissen, ob meine vorgeschlagene Formulierung euch besser gefällt oder ob die jetzige Formulierung ausreichend ist.

---

Ich finde die gut.
Gerne mal machen wenn du Zeit hast in einem neuen PR.